### PR TITLE
Fix: Skeleton UI glitch

### PR DESCRIPTION
**Pull request by feyton**
### PR Description
This pull request fix the issue of unaligned skeleton UI on the dashboard page

### Description of tasks that were expected to be completed
- The skeleton UI for the dashboard has been redesigned to match the content structure
- The loading speed has not been affected (10 seconds of time to action)
- Fix the issue of `misconfigured eslint`
### How has this been tested?
This can be easily tested by logging in and browsing through the dashboard
- The sidebar will not be overlapped by the skeleton loading
- The app speed is still maintained

### Background context
This PR also address the issue of misconfigured `eslint` rules in our project
image

### CODE REVIEW

**Comment by feyton** [comment]
I deployed a review app for you to try out this pull request 👇🏽

Heroku App: https://devpulse-review-app-79.herokuapp.com/


<hidden purpose="for-rewritable-pr-comment-action-use" value="Deploy"></hidden>

**Review by chanceDira — DISMISSED** [review]
Good job @feyton,

I just found an issue in the dashboard, there is some space remaining at the bottom of the page while the skeleton is loading. Take a look at the video below.
 
https://user-images.githubusercontent.com/67952319/187861749-b8a6ed1b-788b-4b63-b150-e212c774f622.mp4

**Comment by feyton** [comment]
> Good job @feyton,
> 
> I just found an issue in the dashboard, there is some space remaining at the bottom of the page while the skeleton is loading. Take a look at the video below.
> WhatsApp.Video.2022-09-01.at.9.51.20.AM.mp4

The issue has been resolved

**Review by jeandamascyifuzo — APPROVED** [review]
nice job

**Review by chanceDira — APPROVED** [review]
Nice work @feyton.

File: .eslintrc [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "plugins": ["react", "@typescript-eslint"],
   "ignorePatterns": ["/root/webpack.config.js"],
   "rules": {
-    "no-console": "error",
+    "no-console": ["error"],
```
Comment (ceelogre) [inline .eslintrc:19]: Is there any difference when you make it an array.

**Review by ceelogre — APPROVED** [review]
It looks like you touched a lot of files that are not related to this task though.

File: .eslintrc (hunk 1 above)
Comment (feyton) [reply .eslintrc:19]: 1. The issue was that the `eslint` was being initialized to run on the files that are located in the parent directory, not traversing the whole directory tree. Reason for many affected files.

2. Making it array ensures that we are rasing the error but give the flexibility to allow some consoles like `console.warn` which can be used outside debugging but to output some information intended for some users (Though I see rare cases of us using these)

**Pull request merged**